alws/crud/sign_task.py

```diff
@@ -402,6 +407,15 @@ async def complete_gen_key_task(
         payload.file_name,
     )
     roles = [r for r in gen_key_task.product.team.roles if 'signer' in r.name]
+
+    await db.execute(
+        update(models.SignKey).where(
+            and_(models.SignKey.product_id == gen_key_task.product.id,
+                 models.SignKey.active)
```

Comment:
Don't we need to explicitly compare `models.SignKey.active` to `True`?

Reply:
I don't believe so, models.SignKey.active loaded as a boolean value so you don't need to explicitly check. Additionally, I've tested and verified the behavior, when a key is added only the active keys get archived and the archived one's don't get re-archived. (I verified this by checking the db and the archive times which don't change).

Reply:
Okay, I didn't know that it could work in that way